### PR TITLE
[0.9.1]support fa3 quant

### PR DESCRIPTION
**Pull request by 22dimensions**
1. move fa3 quant from mindie turbo to vllm-ascend
2. remove mindie turbo quantizer since all quantization method is inside vllm-ascend

### CODE REVIEW

File: vllm_ascend/quantization/faquant.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+from typing import List
+
+import torch
+import torch_npu
+
+from .quant_utils import (SRC_DTYPE_TO_ACL_DTYPE, TYPE_QUANT_QKV_ONLINE,
+                          quant_per_tensor)
+
+
+class AscendFAQuantAttentionMethod:
```
Comment (ApsarasX) [inline vllm_ascend/quantization/faquant.py:27]: Does this feature currently affect accuracy?

**Comment by Yikun** [comment]
Close this because fa3 quant is not planed in v0.9.1, feel free to open this in main
